main.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+define('IS_PHP_THE_BEST', true);
+define ('STAR', '*');
+
+class KabalaException extends Exception {
+    public function __construct() {
+        parent::__construct('Value must be greater than 0');
```

Comment:
Exception should throw sth like 'Kabala must be greater than...'

Reply:
Done in 096cefd